src/actions/mediaActions.js

```diff
@@ -35,17 +41,21 @@ export const findMedia = (referenceValue, institutionId) => (dispatch, getState)
   }
 
   const url = `${gatewayUrl}/es/doc/_search`;
-  const referenceField = config.get('referenceField');
+
+  const sortField = Object.keys(config.get('mediaSnapshotSort'))[0];
+  const sortDirection = config.get('mediaSnapshotSort')[sortField];
 
   const query = {
-    _source: ['collectionspace_denorm:mediaCsid', 'collectionspace_denorm:mediaAltText', 'collectionspace_denorm:title'],
+    _source: ['media_common:blobCsid', 'media_common:altText'],
```

Comment:
I noticed I wasn't seeing any images when testing this locally, and I believe this should use `ecm:name` instead of `blobCsid` in order to get the derivatives back. The API call (which the gateway proxies) looks like `/cspace-services/media/${mediaCsid}/blob/derivatives/${size}/content`.